astro/docs-url-map.json

```diff
@@ -27,7 +27,7 @@
   "/docs/v1/tech/apis/identity-providers/twitter": "/docs/apis/identity-providers/twitter",
   "/docs/v1/tech/apis/identity-providers/xbox": "/docs/apis/identity-providers/xbox",
   "/docs/v1/tech/apis/messengers/generic": "/docs/apis/messengers/generic",
-  "/docs/v1/tech/apis/two-factor": "/docs/apis/messengers/two-factor",
+  "/docs/v1/tech/apis/two-factor": "/docs/apis/two-factor",
```

Comment:
You don't have a redirect for `/docs/apis/messengers/two-factor`=> `/docs/apis/two-factor` which is one of my main concerns. So I'd want to have that added back in.

Reply:
What is the difference between `docs-url-map.json` and `redirects.json`?

In `redirects.json` I do have this accounted for, so are these redundant and I need to add redirects to both, or is one just a map used for something else?

`redirects.json`

```
"/docs/v1/tech/apis/two-factor": "/docs/apis/two-factor",
"/docs/apis/messengers/two-factor": "/docs/apis/two-factor",
```

Reply:
My apologies, I was looking at the wrong file. Your redirects look great.

I don't have any idea what docs-url-map.json is.

Reply:
Resolving.